docs/reference/query-dsl/text-expansion-query.asciidoc

```diff
@@ -87,20 +87,17 @@ is an optimization technique used by {es} to skip documents that cannot score
 competitively against the current best matching documents. However, the tokens
 generated by the ELSER model don't work well with the Max WAND optimization.
 Consequently, enabling Max WAND can actually increase query latency for
-`text_expansion`. For datasets of a significant size, disabling Max
-WAND leads to lower query latencies.
-
+`text_expansion`.
 Max WAND is controlled by the
-<<track-total-hits, track_total_hits>> query parameter. Setting track_total_hits
-to true forces {es} to consider all documents, resulting in lower query
-latencies for the `text_expansion` query. However, other {es} queries run slower
-when Max WAND is disabled.
+<<track-total-hits, track_total_hits>> query parameter. Setting `track_total_hits`
+to `true` forces {es} to consider all documents, resulting in slower {es} queries overall.
+Consider setting `track_total_hits` to `false` for optimal performance.
 
 If you are combining the `text_expansion` query with standard text queries in a
 compound search, it is recommended to measure the query performance before
 deciding which setting to use.
 
 NOTE: The `track_total_hits` option applies to all queries in the search request
-and may be optimal for some queries but not for others. Take into account the
+and may slow down some queries. Take into account the
 characteristics of all your queries to determine the most suitable
 configuration.
```

Comment:
I was hopeful that we could entirely eliminate the whole `optimizing-text-expansion` section. We're [closing the gap in terms of performance](https://elasticsearch-benchmarks.elastic.co/#tracks/msmarco-passage-ranking/nightly/default/90d) so I'd like to avoid making the perception that `the ELSER model don't work well with the Max WAND optimization.`. It will continue to improve so we don't need to ask users to enable the tracking. It is also quite misleading since the total hits returned by a query expansion isn't really useful.

@tveasey @davidkyle WDYT? Are the performance improvements in the latest releases enough to remove the section entirely?

Reply:
My inclination would be to remove entirely at this point. We're pretty close to parity at this point and we know we are actually likely to benefit from Max WAND with the query optimisation planned for 8.12.

Reply:
Below are the results from the old ELSER weighted terms rally track, comparing 8.10 and 8.11 the gap is much smaller, small enough to justify removing the **Optimizing the search performance of the text_expansion query** section. 

Could we leave a note along the line of "Depending on your data, the text expansion query may be faster with `track_total_hits: false`"




|   Metric   | 8.10 Max WAND Enabled | 8.10 Max WAND Disabled  | 8.11 Max WAND Enabled | 8.11 Max WAND Disabled  |
|----------|------------------|-------------------|----------------------|--------------------|
|   50th percentile latency (ms) |     107.985       |   24.0557  | 30.1243  |  25.8158  |
|  90th percentile latency (ms) |    108.904     |     25.2754   | 30.788 |  26.4184 | 
|  99th percentile latency (ms) |   110.134   |     26.0405   | 31.473 | 27.1658 |

Reply:
Updated, thanks!